modules/integration/tests-ui-integration/src/test/java/org/wso2/carbon/mb/ui/test/dlc/DLCQueueTestCase.java

```diff
@@ -81,18 +99,26 @@ public void init() throws Exception {
      * @throws AndesClientException
      */
     @BeforeClass()
-    public void createDLC() throws AndesClientConfigurationException, NamingException, JMSException,
-                                   IOException, AndesClientException {
+    public void addMessagesToDLCQueue() throws AndesClientConfigurationException, NamingException,
+                                               JMSException, IOException, AndesClientException {
+
+        // Get current "AndesAckWaitTimeOut" system property.
+        defaultAndesAckWaitTimeOut = System.getProperty(AndesClientConstants.
+                                                                ANDES_ACK_WAIT_TIMEOUT_PROPERTY);
+
+        // Setting system property "AndesAckWaitTimeOut" for andes
+        System.setProperty(AndesClientConstants.ANDES_ACK_WAIT_TIMEOUT_PROPERTY, "0");
```

Comment:
This will set andes ack wait timeout to 0. when wait time has set to 0 it will wait 0 seconds to until acknowledgment received before sending the message to dead letter channel. Since we want to add messages to DLC fast without a wait we have set this value to 0. 
Improved the comment in code review pr.